README.md

```diff
@@ -9,6 +9,17 @@ SPDX-License-Identifier: MIT OR Apache-2.0
 
 # CXX-Qt
 
```

Comment:
could we add `[![REUSE status](https://api.reuse.software/badge/github.com/KDAB/cxx-qt)](https://api.reuse.software/info/github.com/KDAB/cxx-qt)` as well like Slint has? :-)

Reply:
Great suggestion. Done